operator/bundle_helpers/patch-csv.py

```diff
@@ -60,7 +60,7 @@ def patch_csv(csv_doc, version, operator_image, first_version, no_related_images
 
     csv_doc['spec']['version'] = version
 
-    if not no_related_images:
+    if related_images_mode in ["downstream", "konflux"]:
```

Comment:
[minor] Perhaps this may read a bit more straightforward:
```suggestion
    if related_images_mode != "omit":
```
Up to you.